### DISCUSSION TITLE
Trade license creation error (Structure Type No)

### DISCUSSION THREAD

**Discussion by amitongraph2023** [egovernments/Digit-Core#267] [DIGIT Platform]
![Screenshot 2023-11-07 184935](https://github.com/egovernments/Digit-Core/assets/129284775/5a4de82c-6aa1-4bec-a32d-2802922b0fb9)
![Screenshot 2023-11-07 185054](https://github.com/egovernments/Digit-Core/assets/129284775/de260aeb-1028-480e-a4b1-9b6e35fada0a)

I have deployed the E-Gov 2.7 on my server. When I try to create trade license after choosing structure type no, I'm getting error something went wrong.

**Comment by pradeepkumarcm-egov** [comment]
We need console logs to debug further. The screen shots not give more details. 

As the next step in the screen, structure types are required. This data is present in the [MDMS](https://github.com/egovernments/egov-mdms-data/blob/DEV/data/pb/common-masters/StructureType.json) level.

We need to upload the rate master for the trade types. Use the below postman scripts to push sample data.
[TL-Calculator.postman_collection.json](https://github.com/egovernments/Digit-Core/files/13320340/TL-Calculator.postman_collection.json)
